src/main/g8/src/test/scala/$package$/service/ItemServiceSpec.scala

```diff
@@ -30,33 +30,33 @@ object ItemServiceSpec extends ZIOSpecDefault:
   def spec = suite("item service test")(
     test("get item id accept long") {
       for
-        found <- assertZIO(getItemById(ItemId(123)))(isSome(equalTo(exampleItem)))
+        found  <- assertZIO(getItemById(ItemId(123)))(isSome(equalTo(exampleItem)))
         mising <- assertZIO(getItemById(ItemId(124)))(isNone)
       yield found && mising
-    }.provideCustomLayer(getItemMock >>> ItemServiceLive.layer),
+    }.provide(getItemMock >>> ItemServiceLive.layer),
```

Comment:
```suggestion
    }.provide(getItemMock, ItemServiceLive.layer),
```
I think we can use the syntax